src/modules/location/index.ts

```
@@ -38,6 +42,14 @@ export class LocationModule {
     options:
       | string
       | {
+          /**
+           * The state to generate the zip code for.
+           *
+           * Only works for locales with postcode_by_state definition. If a locale does not
+           * have a postcode_by_state definition, a random zip code is generated according
+           * to the locale's zip format.
```

IMO this should throw an error instead.